backend/services/openai.go

```diff
@@ -16,11 +16,17 @@ var OPEN_AI_KEY string
 
 func Init() {
 	OPEN_AI_KEY := os.Getenv("OPEN_AI_KEY")
-	OpenAIclient = openai.NewClient(OPEN_AI_KEY)
+	OPEN_AI_SERVER_URL := os.Getenv("OPEN_AI_SERVER_URL")
```

Comment:
We do have `config/config.go` module for this. Let's add a new `OpenAIServerURL` variable there.

Reply:
oh, nice catch. Yes, that would be great!